### PR TITLE
Weno bug fixes

### PR DESCRIPTION
**Pull request by sjpadgett**
- rename class file because of case difference. windows doesn't care but linux does

#### Short description of what this resolves:
weird how I had to rename for git prob a windows thing.

### CODE REVIEW

**Comment by sjpadgett** [comment]
@dataengr my repo

**Comment by sjpadgett** [comment]
Runs on docker easy but this module just seems to need improvements in UX. Merging

File: interface/modules/custom_modules/oe-module-weno/templates/facilities.php [hunk 1]
```diff
@@ -73,10 +73,13 @@ function activateFacility(){
         }
         
         function downloadPharmacies(){
+            if (!window.confirm(jsText("This download may take several minutes. Do you want to continue?"))) {
```
Comment (bradymiller) [inline interface/modules/custom_modules/oe-module-weno/templates/facilities.php:76]: no need to escape literal that goes to dialog in js context. Would translate it via the javascript xl() function though.

**Comment by bradymiller** [comment]
2DV

File: interface/modules/custom_modules/oe-module-weno/templates/facilities.php (hunk 1 above)
Comment (sjpadgett) [reply interface/modules/custom_modules/oe-module-weno/templates/facilities.php:76]: oops you're right and I've forgotten about the neat translate js function. We could start using it more in twig I'd think!

**Pull request merged**